etc/kayobe/ansible/reboot.yml

```diff
@@ -9,9 +9,26 @@
     ansible_user: "{{ bootstrap_user if reboot_with_bootstrap_user | bool else kayobe_ansible_user }}"
     ansible_ssh_common_args: "{{ '-o StrictHostKeyChecking=no' if reboot_with_bootstrap_user | bool else '' }}"
     ansible_python_interpreter: /usr/bin/python3
+    confirm_reboot: False
   tags:
     - reboot
   tasks:
+    - name: Prompt to confirm reboot
```

Comment:
I've not used this before, will it spam for every host if you don't have `run_once` and `delegate_to: localhost`?

Reply:
We set `serial: "{{ lookup('env', 'ANSIBLE_SERIAL') | default(1, true) }}"` at the top of this playbook, so it will go through each host one-by-one and prompt for each. 

> The pause module integrates into async/parallelized playbooks without any special considerations (see Rolling Updates). When using pauses with the serial playbook parameter (as in rolling updates) you are only prompted once for the current group of hosts.

The pause module does support serial, so if we set ANSIBLE_SERIAL=0 it will only prompt once for all hosts.